.github/workflows/canary-integration-test.yml

```diff
@@ -96,7 +98,17 @@ jobs:
         # create `radosNamespace` rados-namespace for `replicapool` rbd data-pool
         kubectl -n rook-ceph exec $toolbox -- rbd namespace create replicapool/radosNamespace
         kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --cephfs-filesystem-name myfs --rbd-data-pool-name replicapool --rados-namespace radosNamespace --cluster-name rookStorage --restricted-auth-permission true
-
+    
+    - name: test the upgrade flag
+      run: |
+        toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
```

Comment:
Let's print auth before and after to validate that the caps are upgraded correctly.

Reply:
yes, done